src/test/java/com/kainos/ea/backend/controllers/JobFamilyControllerTest.java

```diff
@@ -0,0 +1,29 @@
+package com.kainos.ea.backend.controllers;
+
+import com.kainos.ea.backend.models.JobFamily;
+import com.kainos.ea.backend.services.JobFamilyService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.*;
+@ExtendWith(MockitoExtension.class)
+class JobFamilyControllerTest {
+
+    @Mock
+    private JobFamilyService jobFamilyService;
+
+    @Test
+    public void whenQueryingDataJobs_expect_ServiceCalledPassback() {
```

Comment:
Could you add an underscore after `when` so it is consistent with our tests? 
`when_QueryingDataJobs_expect_ServiceCalledPassback()`